packages/frontend/src/components/HoursWithdrawal.vue

```diff
@@ -223,9 +222,7 @@ export default Vue.extend({
           rate: transaction.transaction.rate
             ? `${transaction.transaction.rate * 100}%`
             : "",
-          sum: transaction.transaction.rate
-            ? transaction.transaction.hours * transaction.transaction.rate
-            : undefined,
+          sum: this.getTranslatedTotal(transaction.type, transaction),
```

Comment:
```suggestion
          sum: this.getTranslatedTotal(transaction),
```
Ref forslag til omskriving av getTranslatedTotal